CHANGELOG.md

```diff
@@ -21,102 +21,224 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
+* The `construct()` method of the (bi_)fm_index is now private:
   Use the constructor `seqan3::fm_index::fm_index(text_t && text)` or `seqan3::bi_fm_index::bi_fm_index(text_t && text)`
-  instead.
+  instead
+  ([\#1222](https://github.com/seqan/seqan3/pull/1222)).
 
-* **The `seqan3::fm_index::char_type` member was renamed to `seqan3::fm_index::alphabet_type`**
-  The same applies for the `seqan3::bi_fm_index`.
+* The `seqan3::fm_index::char_type` member was renamed to `seqan3::fm_index::alphabet_type`
+  The same applies for the `seqan3::bi_fm_index`
+  ([\#1433](https://github.com/seqan/seqan3/pull/1433)).
 
-* **The `seqan3::fm_index_cursor::index_char_type` member was renamed to `seqan3::fm_index_cursor::index_alphabet_type`**
-  The same applies for the `seqan3::bi_fm_index_cursor`.
+* The `seqan3::fm_index_cursor::index_char_type` member was renamed to
+  `seqan3::fm_index_cursor::index_alphabet_type`
+  The same applies for the `seqan3::bi_fm_index_cursor`
+  ([\#1433](https://github.com/seqan/seqan3/pull/1433)).
 
 ## Notable Bug-fixes
 
-* Copying and moving the `seqan3::fm_index` and `seqan3::bi_fm_index` now work properly.
-* Searching in the `seqan3::fm_index` and `seqan3::bi_fm_index` constructed from a text collection containing a single
-  text now return correct result.
-* Translation table for nucleotide to amino acid translation now corrected.
+* All our headers are self contained
+  ([\#1085](https://github.com/seqan/seqan3/pull/1085)).
+* The alignment algorithm with edit distance returns the correct back coordinate
+  ([\#1093](https://github.com/seqan/seqan3/pull/1093)).
+* Inserting or deleting gaps into an empty `seqan3::gap_decorator` does not cause assert anymore
+  ([\#1109](https://github.com/seqan/seqan3/pull/1109)).
+* Some fixes to edge cases in BAM file writing
+  ([\#1110](https://github.com/seqan/seqan3/pull/1110)).
+* The application name of the `seqan3::argument_parser` is restricted to alpha-numeric characters and `_` and `-`
+  ([\#1133](https://github.com/seqan/seqan3/pull/1133)).
+* Copying and moving the `seqan3::fm_index` and `seqan3::bi_fm_index` now work properly
+  ([\#1144](https://github.com/seqan/seqan3/pull/1144)).
+* Searching in the `seqan3::fm_index` and `seqan3::bi_fm_index` constructed from a text collection containing a
+  single text now return correct result
+  ([\#1316](https://github.com/seqan/seqan3/pull/1316)).
+* The view `seqan3::views::take` is sized if the underlying range is sized
+  ([\#1146](https://github.com/seqan/seqan3/pull/1146)).
+* The detection of the pthread library works correctly on linux based systems
+  ([\#1200](https://github.com/seqan/seqan3/pull/1200)).
+* The translation table for nucleotide to amino acid translation was corrected
+  ([\#1485](https://github.com/seqan/seqan3/pull/1485)).
+* The amino acid score matrices were corrected
+  ([\#1455](https://github.com/seqan/seqan3/pull/1455)).
 
 # 3.0.0 ("Escala")
 
```

Comment:
The part below here should be the same as the description in https://github.com/seqan/seqan3/releases/tag/3.0.0

In general, the text in the releases should be the same as the changelog.

```
This is the initial release of SeqAn3. It is an entirely new library so there is no changelog that covers the differences to SeqAn2.

Please see the release announcement:
https://www.seqan.de/announcing-seqan3/

See the porting guide for some help on porting:
http://docs.seqan.de/seqan/3-master-user/howto_porting.html

Note that 3.1.0 will be the first API stable release and interfaces in this release might still change.
```

Reply:
I would also add a link to the starting tutorial for SeqAn 3